policy/executor/internal/nodes.go

```diff
@@ -347,7 +360,16 @@ func (nodeData *ReportingQueryNodeData) score() *policy.Score {
 	}
 
 	if allFound {
-		if foundError {
+		if assetVanishedDuringScan {
```

Comment:
In general, a suggestion: 
I feel like we should be able to mark assets themselves as ephemeral (can tie either with asset type or provider type maybe?) and act accordingly. 
I think a better way to fix this would be when starting the scan: if the asset does not exist then, we should be able to report accordingly upstream.

Reply:
I totally agree it's a bigger topic, for sure not something that can be done quickly.

I was thinking more along the lines of having a specific error type that can be thrown if the resource does not exist rather than relying on the message inside the error

Reply:
I implemented a k8s specific error type in cnquery and check for it.